insights/config/specs.py

```diff
@@ -395,6 +395,8 @@
     "systemid"                  : First([SimpleFileSpec("etc/sysconfig/rhn/systemid"),
                                     SimpleFileSpec("conf/rhn/sysconfig/rhn/systemid")]),
     "teamdctl_state_dump"       : CommandSpec("/usr/bin/teamdctl {iface} state dump", iface=r"team\S+"),
+    "thp_use_zero_page"         : SimpleFileSpec("sys/kernel/mm/transparent_hugepage/use_zero_page"),
+    "thp_enabled"               : SimpleFileSpec("sys/kernel/mm/transparent_hugepage/enabled"),
```

Comment:
You may forget to add the two specs to "insights/specs.py", it's necessary for the master branch.

Reply:
Thank you, I was not sufficiently aware of the new specs.